Source/EventHorizon/Producer/Filter/PublicFilterDefinition.cs

```diff
@@ -27,5 +27,8 @@ public PublicFilterDefinition(StreamId sourceStream, StreamId targetStream)
 
         /// <inheritdoc/>
         public StreamId TargetStream { get; }
+
+        /// <inheritdoc/>
+        public bool IsPersistable => false;
```

Comment:
You could have an interface `ICanBePersisted` instead - marker interface. Use this for checking if it should be persisted (`if (filterDefinition is ICanBePersisted) {}`)

Reply:
Or...  `IPersistedFilterDefinition`